### DISCUSSION TITLE
Created a new pipeline for MoA (Mixture of Agents) for Groq .  Works really great.  I like to contribute it to this community

### DISCUSSION THREAD

**Discussion by raymondbernard** [open-webui/pipelines#155] [Ideas]
Please take a look at my video and blog regarding the feature.  
It does outperform the state-of-the-art models like GPT 4.0  Let me know how we sould proceed to contribute it .  
I do by the way use a .env to store the details for keys and URL of the agents\models. Here is a link to my blog: raymondbernard@github.io and the YouTube video is https://www.youtube.com/watch?v=KxT7lHaPDJ4&t=371s

**Comment by Derjyn** [comment]
The link to your blog is an email (`mailto`), and the link to the video has a `t=` timestamp at 371 seconds. Might want to remedy that, and also tune up your pipeline script and `.env`; It seems you got started on Ollama integration, which would be _very_ awesome.

It's definitely a cool pipeline, and I personally appreciate it! It can be tricky getting an MoA/MoE up and running, so it's always great to see these resources pop up. Keep at it!
